src/NorthwindWebsite/Presentation/Views/Products/FromApiJS.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿<h1>Products from API with Javascript</h1>
+
+<script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-2.0.3.min.js"></script>
+<link rel="stylesheet" href="~/css/entities-from-api.css" />
+
+<div>
+    <ul id="products" />
+</div>
+
+<script>
+    let $productsList = $('#products');
+    let apiUrl = '../api/products/';
+    let method = 'GET';
+
+    $(function(){
+        $.ajax({
+        type: method,
+        url: apiUrl,
+        success: function(products) {
+           $.each(products, function(i, product){
```

Comment:
Honestly, I am not sure that JQuery `each` loop is required here since you are iterating API response, not DOM-items. Let's replace it with `for (let product of products)` loop, please